dbms/src/Functions/FunctionHelpers.h

```diff
@@ -24,6 +24,17 @@ const Type * checkAndGetDataType(const IDataType * data_type)
     return typeid_cast<const Type *>(data_type);
 }
 
+template <typename Type>
+std::shared_ptr<const Type> checkAndGetDataTypePtr(const DataTypePtr & data_type)
```

Comment:
I can't find where this function is used. If it isn't, let's remove it.

Reply:
Done